client/src/pages/LandingPage.js

```diff
@@ -57,68 +55,48 @@ const LandingPage = () => {
   ];
 
   return (
-    <div className="bg-primary flex justify-center">
-      <div className="w-full tablet:w-11/12 desktop:w-2/3">
-        <div className="flex flex-wrap desktop:w-full pt-20 tablet:pt-0">
-          <HamburgerNav logo={"./logoicon.png"} logotext={"./logotext.png"} />
-          <NavContainer />
-          <LongCard
-            img={"/Homeoffice.png"}
-            heading="Welcome"
-            title="Together"
-            description=" is a new app designed for
+    <div className="flex flex-col items-center w-full pt-20 min-h-screen pb-10 md:pt-5 lg:text-lg bg-primary">
+      <HamburgerNav logo={"./logoicon.png"} logotext={"./logotext.png"} />
+      <div className="flex flex-col items-center gap-10 w-full p-4 max-w-xl md:max-w-3xl lg:max-w-5xl lg:gap-20 xl:max-w-7xl">
+        <NavContainer />
+        <LongCard
+          img={"/Homeoffice.png"}
+          heading="Welcome"
+          title="Together"
+          description=" is a new app designed for
               100Devs. Together is a group calendar for public events, so you can
               see what's going on in your community. We're making it
               easier to find out about the events that are happening in Discord and
               let you know how much fun they'll be!"
-          />
-          <div className="w-full p-6 text-center tablet:inline-flex tablet:space-x-10 tablet:p-6 tablet:w-full tablet:h-[27rem]">
-            {cards.map((cardData, index) => {
-              return (
-                <SmallCard
-                  heading={cardData.heading}
-                  description={cardData.description}
-                  Icon={CgChevronDoubleRightO}
-                  key={index}
-                  index={index}
-                />
-              );
-            })}
-          </div>
-          <BottomCard
-            heading="Who we are"
-            description={
-              <>
-                <p>
```

Comment:
This `p` is causing a `Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.` warning. The `BottomCard` already wraps the description in a `p` so it could have been removed.